test/packageLogs_spec.js

```diff
@@ -14,356 +14,316 @@
-          const evt = new window.MouseEvent('click', {
-            'view': window,
-            'bubbles': true,
-            'cancelable': true
-          });
-          document.body.appendChild(ele);
-          ele.addEventListener('click', (e) => {
-            document.documentElement.scrollLeft = 0;
-            document.documentElement.scrollTop = 0;
-            const originalDocument = global.document;
-            global.document = document;
-            expect(getLocation(e)).to.deep.equal({ x: 0, y: 0 });
-            global.document = originalDocument;
-            done();
-          });
-          ele.dispatchEvent(evt);
-        },
-      });
+    describe('getSelector', () => {
+        it('builds a selector', () => {
+            new JSDOM(``)
+            const document = window.document;
+            const element = document.createElement('div');
+            expect(getSelector(element)).to.equal('div');
+            element.id = 'bar';
+            expect(getSelector(element)).to.equal('div#bar');
+            element.removeAttribute('id');
+            element.classList.add('baz');
+            expect(getSelector(element)).to.equal('div.baz');
+            element.id = 'bar';
+            expect(getSelector(element)).to.equal('div#bar.baz');
+        });
     });
 
-    it('fails to null', (done) => {
-      let hadError = false;
-      try {
-        getLocation(null);
-      } catch (e) {
-        hadError = true;
-      }
-      expect(hadError).to.equal(true);
-      done();
-    });
-  });
-
-  describe('selectorizePath', () => {
-    it('returns a new array of the same length provided', (done) => {
-      const arr = [{}, {}];
-      const ret = selectorizePath(arr);
-      expect(ret).to.be.instanceof(Array);
-      expect(ret).to.not.equal(arr);
-      expect(ret.length).to.equal(arr.length);
-      done();
-    });
-  });
-
-  describe('getSelector', () => {
-    it('builds a selector', (done) => {
-      jsdom.env({
-        url, html,
-        done: (err, window) => {
-          const document = window.document;
-          const element = document.createElement('div');
-          expect(getSelector(element)).to.equal('div');
-          element.id = 'bar';
-          expect(getSelector(element)).to.equal('div#bar');
-          element.removeAttribute('id');
-          element.classList.add('baz');
-          expect(getSelector(element)).to.equal('div.baz');
-          element.id = 'bar';
-          expect(getSelector(element)).to.equal('div#bar.baz');
-          done();
-        },
-      });
+    it('identifies window', () => {
+        new JSDOM(``)
+        expect(getSelector(window)).to.equal('Window');
     });
 
-    it('identifies window', (done) => {
-      jsdom.env({
-        url, html,
-        done: (err, window) => {
-          expect(getSelector(window)).to.equal('Window');
-          done();
-        },
-      });
+    it('handles a non-null unknown value', () => {
```

Comment:
making minor edit here to pull 'identifies window' into describe('getSelector')

Reply:
Merged @UncleGedd's generous contribution. Note: pushed minor mods to resolve minor issue of orphaned getSelector tests.